src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const _ = require('lodash');
+
+/* global sanitizeHtml: false */
+angular.module('esn.inbox.libs').filter('sanitizeStylisedHtml', function ($sce) {
```

Comment:
The variable `$sce` is ambiguous, I don't understand its purpose just by looking at the name.
Can you please use a longer name to make it more clear?

Reply:
It's not a variable that we define. It belongs to AngularJS, like `$q`.

Reply:
i just replaced the file from unified inbox to inbox libs ;)